dviasm.py

```diff
@@ -679,9 +688,8 @@ def WriteFontDefinitions(self, fp):
         s.append(PutSignedQuad(self.font_def[e]['scaled_size']))
         s.append(Put2Bytes(flags))
         s.append(PutByte(len(self.font_def[e]['name'])))
-        s.append(self.font_def[e]['name'])
+        s.append(self.font_def[e]['name'].encode('utf8'))
         s.append(PutSignedQuad(self.font_def[e]['index']))
-        print >> sys.stderr, self.font_def[e]['name'], self.font_def[e]['index']
```

Comment:
This line was originally introduced in [that commit](/khaledhosny/dviasm/commit/c3487d0) over five years ago, it generates a lot of output in some cases, which I don’t think is useful anymore.

Reply:
Right, that shouldn’t have been committed in the first place.